soc/arm/nxp_imx/rt5xx/Kconfig.defconfig.mimxrt595_cm33

```diff
@@ -0,0 +1,41 @@
+# NXP MIMXRT5XX platform configuration options
+
+# Copyright (c) 2022, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_MIMXRT595S_CM33
+
+config SOC
+	default "mimxrt595s_cm33"
+
+config PINMUX_MCUX_LPC
+	default y
+	depends on PINMUX
```

Comment:
note: all pinmux seems to happen at board level, is that really needed?

Reply:
This is so that we can enable the PINMUX driver.